.github/workflows/draw-zmk.yml

```diff
@@ -34,6 +34,11 @@ on:
         default: "[Skip CI] keymap-drawer render"
         required: false
         type: string
+      ammend_commit:
+        description: "Whether to ammend the last commit instead of creating a new one"
```

Comment:
```suggestion
        description: "Whether to amend the last commit instead of creating a new one"
```

Reply:
Looks like my brain just misspells that word everywhere